### PR TITLE
Rename references to tavultesoft.keymanweb in KeymanWeb compiler and debugger

### PR DESCRIPTION
**Pull request by mcdurdin**

### CODE REVIEW

File: windows/src/developer/TIKE/compile/CompileKeymanWeb.pas [hunk 1]
```diff
@@ -2213,8 +2213,9 @@ function TCompileKeymanWeb.JavaScript_SetupProlog: string;
 begin
   if IsKeyboardVersion10OrLater then
   begin
-    Result := 'if(parseFloat(tavultesoft.keymanweb.version) < 10) {'+nl+
-      FTabStop+'tavultesoft.keymanweb.util.alert("This keyboard requires KeymanWeb 10.0 or later");'+nl+
+    Result := 'if(typeof keyman === ''undefined'') {'+nl+
+      FTabStop+'console.log(''Keyboard requires KeymanWeb 10.0 or later'');'+nl+
+      FTabStop+'if(typeof tavultesoft !== ''undefined'') tavultesoft.keymanweb.util.alert("This keyboard requires KeymanWeb 10.0 or later");'+nl+
```
Comment (jahorton) [inline windows/src/developer/TIKE/compile/CompileKeymanWeb.pas]: My one question:  where does this block of code get placed?  Before the `KR`/`registerKeyboard` call, in `gs()`...?

**Comment by jahorton** [comment]
Two other questions - what issue(s) does this fix?
Do we want to put something in history.md about it?

File: windows/src/developer/TIKE/compile/CompileKeymanWeb.pas (hunk 1 above)
Comment (mcdurdin) [reply windows/src/developer/TIKE/compile/CompileKeymanWeb.pas]: This gets placed at the very start of the file, for example:
```
if(typeof keyman === 'undefined') {
  console.log('Keyboard requires KeymanWeb 10.0 or later');
  if(typeof tavultesoft !== 'undefined') tavultesoft.keymanweb.util.alert("This keyboard requires KeymanWeb 10.0 or later");
} else {
KeymanWeb.KR(new Keyboard_version10chiral());
}
function Keyboard_version10chiral()
{
```

**Comment by mcdurdin** [comment]
This PR:

1) Makes sure that keyboards that have v10 features don't load on older versions of KMW;
2) Replaces the reference to `tavultesoft.keymanweb` with `keyman`. These were added in the v10 compiler to support references to `osk.modifierCodes` and `osk.keyCodes`, so there are no back-compat issues.

**Pull request merged**